theses_base.html

```diff
@@ -0,0 +1,216 @@
+{% extends "base.html" %}
+
+{% load wagtailcore_tags %}
+{% load wagtailimages_tags %}
+{% load i18n %}
+{% load menu_tags %}
+{% load static %}
+{% load wagtailuserbar %}
+
+{% block extra_css %}
+    <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet">
+    <link rel="stylesheet" type="text/css" href="{% static 'css/eacr.css' %}">
+    <link rel="stylesheet" type="text/css" href="{% static 'css/theses.css' %}">
+<<<<<<< HEAD

```

Comment:
These are leftover merge markers. Please make sure the merge is okay and remove them. (Here and below.)